src/main.js

```diff
@@ -136,6 +137,7 @@ app.component('PvCard', PvCard);
 app.component('PvChart', PvChart);
 app.component('PvCheckbox', PvCheckbox);
 app.component('PvChip', PvChip);
+app.component('PvChips', PvChips);
```

Comment:
Curious why you're preferring to load the component globally instead of locally? Same as above, I'm intending to write a ticket for this for generally speaking, but since this is introducing a new global component, I wanted to flag this here as well.

As I'm new to the project I might be not fully familiar with some of the reasonings behind certain approaches but generally speaking, global component registration is pretty bad [for a few reasons](https://vuejs.org/guide/components/registration#local-registration), one of them being tree-shaking.

Reply:
I agree with @maximilianoertel here. Thanks for bringing it up. I've added a task to the next sprint to prefer local registration over global registration and assigned it to you @maximilianoertel. I think it's a really good point that you bring up but I don't think it should block this PR.